holopy/scattering/imageformation.py

```diff
@@ -0,0 +1,233 @@
+import numpy as np
```

Comment:
Hmm... let me think about if there is a better way to do this. The old logs are still accessible via `git log scatteringtheory.py`. I will see if there is a way to link the files in the git history manually, or if I need to do a rebase / new pr with cherry-picks of the commits. 

I will address this last though so you can see the changes and reply to my comments. LMK when you are OK with the other comments, and I will address this one.

Reply:
I had done something similar to what you suggested when originally doing this. So while `git log imageformation.py` only returns 5 commits, `git log --follow imageformation.py` returns all 127 commits. 

Let me know if you think that is sufficient.

Reply:
It would be nice to have the blame reflect earlier history, e.g. at https://github.com/briandleahy/holopy/blame/composescatteringtheory/holopy/scattering/imageformation.py

I think you should be able to just copy paste the content from this PR into 2 new branches, but if it will be a huge pain we can leave it as is...